index.html

```diff
@@ -32,7 +32,8 @@
           <header>
             <h1 class="Name">Javed Shahbaz</h1>
           </header>
-          <p class="headLine">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+          <p class="headLine">Hello! I am a software developer!
+            I can help you build a product, feature or website. Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
           <button class="blackButton">
```

Comment:
- [ ] The about section too has some isue to fix. Please see the image below 👇🏾 

- The section highlighted with a red square and letter A is missing some illustration
- The button - `get my CV` does not match the Figma design. Please fix.

Your design
![image](https://user-images.githubusercontent.com/84156714/221374292-9b490218-8c4b-4587-951a-db19eda820d4.png)

Figma Design
![image](https://user-images.githubusercontent.com/84156714/221374323-9ed944bf-7e82-4551-9ff3-c2500850eaa7.png)

Reply:
Fixed.